.gitignore

```diff
@@ -25,3 +25,4 @@ temp/
 
 logs/
 __pycache__/
+website/agents.md
```

Comment:
This looks like it was committed by mistake?

Reply:
THanks - yes this was a hack before another thing was merged